eras/babbage/impl/testlib/Test/Cardano/Ledger/Babbage/Imp/UtxowSpec/Valid.hs

```diff
@@ -10,14 +9,37 @@
 
 module Test.Cardano.Ledger.Babbage.Imp.UtxowSpec.Valid (spec) where
 
+import Cardano.Ledger.Alonzo.TxWits (unTxDatsL)
 import Cardano.Ledger.Babbage.Core
 import Cardano.Ledger.BaseTypes
-import Cardano.Ledger.Coin (Coin (..))
-import Cardano.Ledger.Shelley.Scripts (pattern RequireAnyOf)
-import Cardano.Ledger.TxIn (mkTxInPartial)
+import Cardano.Ledger.Credential (Credential (..), StakeReference (..))
+import Cardano.Ledger.Plutus (
+  Data (..),
+  Datum (..),
+  Language (..),
+  hashData,
+  hashPlutusScript,
+  mkInlineDatum,
+  withSLanguage,
+ )
+import Cardano.Ledger.Shelley.Scripts (
+  ShelleyEraScript,
+  pattern RequireAllOf,
+  pattern RequireAnyOf,
+ )
 import Lens.Micro
+import Lens.Micro.GHC ()
+import qualified PlutusLedgerApi.V1 as PV1
 import Test.Cardano.Ledger.Alonzo.ImpTest
+import Test.Cardano.Ledger.Core.Utils (txInAt)
 import Test.Cardano.Ledger.Imp.Common
+import Test.Cardano.Ledger.Plutus.Examples
+
+nativeSucceeds :: forall era. ShelleyEraScript era => Script era
+nativeSucceeds = fromNativeScript @era $ RequireAllOf []
+
+nativeFails :: forall era. ShelleyEraScript era => Script era
+nativeFails = fromNativeScript @era $ RequireAnyOf []
```

Comment:
It would be nice to move these helpers into the package where they can start being used:
`cardano-ledger-shelley:testlib:Test.Cardano.Ledger.Shelley.Era` seems like a nice place

Also recording such addition in the `testlib` section of the changelog is customary.

Also type application isn't needed.

Also, for consistency sake with plutus helpers like these I'd like to suggest renaming them:
```suggestion
nativeAlwaysSucceeds :: ShelleyEraScript era => Script era
nativeAlwaysSucceeds = fromNativeScript $ RequireAllOf []

nativeAlwaysFails :: ShelleyEraScript era => Script era
nativeAlwaysFails = fromNativeScript $ RequireAnyOf []
```